emily/handler/src/database/accessors.rs

```diff
@@ -204,7 +165,7 @@ pub async fn get_withdrawal_entries(
 }
 
 /// Wipes all the tables.
-/// TODO(TBD): Include check for whether the table is running locally.
+/// TODO(395): Include check for whether the table is running locally.
```

Comment:
That's a good further guard to prevent this from accidentally falling into any logic outside the `testing` feature! That said, it still leaves the path where we accidentally compile the lambda with the testing flag. The lambda should be able to do a quick check to see if it's in production.

Reply:
> That's a good further guard to prevent this from accidentally falling into any logic outside the `testing` feature! That said, it still leaves the path where we accidentally compile the lambda with the testing flag. The lambda should be able to do a quick check to see if it's in production.

Oh yeah good point.